builtin/providers/aws/cloudfront_distribution_configuration_structure_test.go

```diff
@@ -417,7 +443,7 @@ func TestCloudFrontStructure_expandTrustedSigners_empty(t *testing.T) {
 	data := []interface{}{}
 	ts := expandTrustedSigners(data)
 	if *ts.Quantity != 0 {
-		t.Fatalf("Expected Quantity to be 2, got %v", *ts.Quantity)
+		t.Fatalf("Expected Quantity to be 0, got %v", *ts.Quantity)
```

Comment:
I found two others and put up #10997 to fix them. I can drop this change from this PR if that would make things easier...?